src/main/java/org/wso2/lsp4intellij/client/DefaultLanguageClient.java

```diff
@@ -338,6 +335,7 @@ public void notifyProgress(ProgressParams params) {
                 title = progressNotificationItems.get(token).getFirst();
             }
         }
+        String extension = ((ServerWrapperBaseClientContext) context).getWrapper().serverDefinition.ext;
```

Comment:
We might need to do an `instanceof` check before directly casting to`ServerWrapperBaseClientContext`.